code/datums/uplink_item.dm

```diff
@@ -1127,45 +1108,43 @@ var/list/uplink_items = list()
 	desc = "An implant injected into another body, forcing the victim to obey any command by the user for around 15 to 20 mintues."
 	excludefrom = list(/datum/game_mode/nuclear)
 	item = /obj/item/weapon/storage/box/syndie_kit/imp_mindslave
-	cost = 9
+	cost = 8
 	surplus = 20
 
 /datum/uplink_item/implants/greatermindslave
 	name = "Greater Mindslave Implant"
 	desc = "An implant injected into another body, forcing the victim to obey any command by the user, it does not expire like a regular mindslave implant."
 	item = /obj/item/weapon/storage/box/syndie_kit/imp_gmindslave
-	cost = 16
+	cost = 14
```

Comment:
Greater mind slave still sees play from time to time moreso than it's temporary cousin. I think keeping at 16 is better considering who you can implant.